src/me.rs

```diff
@@ -56,10 +56,10 @@ pub fn motion_estimation(
       let border_h = 128 + blk_h as isize * 8;
       let cols = (rec.frame.planes[0].cfg.width + MI_SIZE - 1) / MI_SIZE;
       let rows = (rec.frame.planes[0].cfg.height + MI_SIZE - 1) / MI_SIZE;
-      let x_min = -(bo.x as isize) * (8 * MI_SIZE) as isize - border_w;
-      let x_max = (cols - bo.x - blk_w / MI_SIZE) as isize * (8 * MI_SIZE) as isize + border_w;
-      let y_min = -(bo.y as isize) * (8 * MI_SIZE) as isize - border_h;
-      let y_max = (rows - bo.y - blk_h / MI_SIZE) as isize * (8 * MI_SIZE) as isize + border_h;
+      let x_min = -(bo.x as isize) * (8 * MI_SIZE) as isize - border_w + pmv.col as isize;
+      let x_max = (cols - bo.x - blk_w / MI_SIZE) as isize * (8 * MI_SIZE) as isize + border_w +  pmv.col as isize;
```

Comment:
This doesn't look right. pmv should be added to x/y_lo/hi instead of x/y_min/max to move the search center

Reply:
Oh, right. Ha.. nothing is easy... will fix soon.